mws/apis/reports.py

```diff
@@ -309,3 +314,21 @@ class ReportType(Enum):
     FBA_REMOVAL_RECOMMENDED = '_GET_FBA_RECOMMENDED_REMOVAL_DATA_'
     FBA_REMOVAL_ORDER_DETAIL = '_GET_FBA_FULFILLMENT_REMOVAL_ORDER_DETAIL_DATA_'
     FBA_REMOVAL_SHIPMENT_DETAIL = '_GET_FBA_FULFILLMENT_REMOVAL_SHIPMENT_DETAIL_DATA_'
+
+
+class Schedule(Enum):
+    MINUTES_15 = '_15_MINUTES_'
+    MINUTES_30 = '_30_MINUTES_'
+    HOURS_1 = '_1_HOUR_'
+    HOURS_2 = '_2_HOURS_'
+    HOURS_4 = '_4_HOURS_'
+    HOURS_8 = '_8_HOURS_'
+    HOURS_12 = '_12_HOURS_'
+    DAYS_1 = '_1_DAY_'
+    DAYS_2 = '_2_DAYS_'
+    DAYS_3 = '_72_HOURS_'
+    WEEKS_1 = '_1_WEEK_'
+    DAYS_14 = '_14_DAYS_'
+    DAYS_15 = '_15_DAYS_'
+    DAYS_30 = '_30_DAYS_'
+    DELETE = '_NEVER_'
```

Comment:
Good addition of the enum here, similar to the `ReportType` enumeration. Once we finally get to writing proper documentation, we'll need to be sure to expose these more for end users.

Optionally we should include links to the MWS docs for those enumerations as docstrings ([link for `Schedule` enumerations](http://docs.developer.amazonservices.com/en_US/reports/Reports_Schedule.html)). Not a show-stopper here, though.